tests/test_nuancier.py

```diff
@@ -1849,6 +1849,7 @@ def test_update_candidate(self):
                 in output.data.decode('utf-8'))
 
         upload_path = os.path.join(PICTURE_FOLDER, 'F21')
+        print(upload_path)
```

Comment:
You probably forgot to delete this print.

Reply:
I did, thanks :)